docs/develop/api-reference/auth-tokens.md

```diff
@@ -0,0 +1,181 @@
+---
+sidebar_position: 7
+title: Auth control token API reference information
+sidebar_label: Auth control tokens
+description: Learn the auth control token API calls you need to know about and how to use them with Momento services.
+---
+
+import { SdkExampleTabs } from "@site/src/components/SdkExampleTabs";
+// This import is necessary even though it looks like it's un-used; The inject-example-code-snippet
+// plugin will transform instances of SdkExampleTabs to SdkExampleTabsImpl
+import { SdkExampleTabsImpl } from "@site/src/components/SdkExampleTabsImpl";
+
+# Auth control API reference
+
+<img src="/img/access-tokens.jpg" width="90%" alt="a technical illustration of a bank vault representing security, authorization, and authentication." />
+
+These APIs are used to manage Momento auth tokens and access. Tokens can be scoped to have one or more permissions to give access to one or more caches or topics.
```

Comment:
```suggestion
These APIs are used to manage Momento access control tokens and access. Tokens can be scoped to have one or more permissions to give access to one or more caches or topics.
```

Reply:
ok, this one is a problem. most everywhere in the docs we call these tokens "Momento auth tokens", not Momento access control tokens. Has this changed and we need to change this everywhere?

Reply:
We cannot get rid of the `auth token` terminology. It is everywhere including api names. I suggest we add a transition phrase like `Momento authentication and authorization token for fine grained access control, henceforth referred to as auth token` at the beginning of the page.